npm/src/language-service/language-service.ts

```diff
@@ -60,6 +61,7 @@ export interface ILanguageService {
 export type ILanguageServiceWorker = ILanguageService & IServiceProxy;
 
 export class QSharpLanguageService implements ILanguageService {
+  private qsharpLibraryUriScheme = "qsharp-library-source";
```

Comment:
It is defined in [common.ts](https://github.com/microsoft/qsharp/blob/main/vscode/src/common.ts), but that is located in the vscode extension, which depends on the npm, so the npm cannot take a dependencies on the extension to get access to that. If I find a more public-facing place in the npm to put this, then maybe I can have the extension get it from the npm rather than defining it itself.

Reply:
I've moved the definition that the vscode extension uses to use the npm one so it is more centralized.